health-services/product/src/main/resources/product-persist.yml

```diff
@@ -0,0 +1,89 @@
+serviceMaps:
```

Comment:
We seem to have 2 persister files with some duplicate content. Can we consolidate everything into one file `product-persister.yml`?

Reply:
product-persist.yml file had all the consolidated changes, but I now I have kept only one file `product-persister.yml` which contains insert and updates for both product and product-variant.